website/static/js/osfHelpers.js

```diff
@@ -734,6 +734,23 @@ function indexOf(array, searchFn) {
     return -1;
 }
 
+/**
+ * Check if any of the values in an array are truthy
+ *
+ * @param {Array[Any]} listOfBools
+ * @returns {Boolean}
+ **/
+var any = function(listOfBools) {
```

Comment:
Depending on how general we want this function to be, underscore.js provides a possibly more generic implementation.
http://underscorejs.org/docs/underscore.html#section-31